core/server/master/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java

```diff
@@ -65,7 +64,7 @@ public BlockMasterClientRestServiceHandler(@Context ServletContext context) {
    */
   @GET
   @Path(SERVICE_NAME)
-  @ReturnType("java.lang.String")
+  //@ReturnType("java.lang.String")
```

Comment:
Currently commenting these out because the return type will be reused when we reannotate with Swagger's annotations.